es/03_5_Entendiendo_El_Descriptor.md

```diff
@@ -0,0 +1,132 @@
+
+`xpubs` y` xprvs` resultaron ser insuficientes cuando los tipos de claves públicas se multiplicaron bajo la [expansión SegWit](https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line/blob/master/04_6_Creating_a_Segwit_Transaction .md), de ahí la necesidad de los "descriptores de salida".
+
+> :book: ***¿Qué es un descriptor de salida?*** Una descripción precisa de cómo derivar una dirección de Bitcoin a partir de una combinación de una función y una o más entradas a esa función.
+
+La introducción de funciones en los descriptores es lo que los hace poderosos, porque se pueden usar para transferir todo tipo de direcciones, desde las direcciones heredadas con las que estamos trabajando ahora hasta las direcciones Segwit y multifirma que encontraremos más adelante. Una función individual coincide con un tipo particular de dirección y se correlaciona con reglas específicas para generar esa dirección.
+
+## Capturar un descriptor
+
+Los descriptores son visibles en varios comandos como `listunspent` y`getaddressinfo`:
+
+```
+$ bitcoin-cli getaddressinfo ms7ruzvL4atCu77n47dStMb3of6iScS8kZ
+{
+  "address": "ms7ruzvL4atCu77n47dStMb3of6iScS8kZ",
+  "scriptPubKey": "76a9147f437379bcc66c40745edc1891ea6b3830e1975d88ac",
+  "ismine": true,
+  "solvable": true,
+  "desc": "pkh([d6043800/0'/0'/18']03efdee34c0009fd175f3b20b5e5a5517fd5d16746f2e635b44617adafeaebc388)#4ahsl9pk",
+  "iswatchonly": false,
+  "isscript": false,
+  "iswitness": false,
+  "pubkey": "03efdee34c0009fd175f3b20b5e5a5517fd5d16746f2e635b44617adafeaebc388",
+  "iscompressed": true,
+  "ischange": false,
+  "timestamp": 1592335136,
+  "hdkeypath": "m/0'/0'/18'",
+  "hdseedid": "fdea8e2630f00d29a9d6ff2af7bf5b358d061078",
+  "hdmasterfingerprint": "d6043800",
+  "labels": [
+    ""
+  ]
+}
+```
+
+Aquí el descriptor es `pkh ([d6043800 / 0 '/ 0' / 18 '] 03efdee34c0009fd175f3b20b5e5a5517fd5d16746f2e635b44617adafeaebc388) # 4ahsl9pk`.
+
+## Entender un descriptor
+
+Un descriptor se divide en varias partes:
+
+```
+function([derivation-path]key)#checksum
+```
+
+Esto es lo que todo eso significa:
+* **Función.** La función que se utiliza para crear una dirección a partir de esa tecla. En estos casos, es `pkh`, que es la dirección heredada P2PKH estándar que conoció en [§3.3: Configuración de su billetera](03_3_Setting_Up_Your_Wallet.md). De manera similar, una dirección P2WSH SegWit usaría `wsh` y una dirección P2WPKH usaría` wpkh`.
```

Comment:
- `La función que se utiliza para crear una dirección a partir de esa tecla.` ---> `La función que se utiliza para crear una dirección a partir de esa llave/clave.`
- `[3.3: Configurando Su Billetera](03_3_Configurando_Su_Billetera.md)`

Reply:
- Missing changes. Both are important.
- The use of `tecla` instead of `llave` changes the meaning of the sentence.
- In the second bullet point, the link won't work.

Reply:
- `La función que se utiliza para crear una dirección a partir de esa tecla.` ---> `La función que se utiliza para crear una dirección a partir de esa llave/clave.` The use of `tecla` instead of `llave` changes the meaning of the sentence.
- `[3.3: Configurando Su Billetera](03_3_Configurando_Su_Billetera.md)`. The link as it is won't work.